proto/chart/v1/config.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+option go_package = "chart/config/v1;charts";
+
+package chart.v1;
```

Comment:
```suggestion
option go_package = "chart/config/v1;config";

package config.v1;
```

```suggestion
option go_package = "chart/config/v1;chartconfig";

package config.v1;
```

Reply:
Thanks for the suggestions @lvrach ! I think the first one looks a bit better to me